css/positioning/index.css

```diff
@@ -18,4 +18,6 @@
 	top: 20px;
 	left: 450px;
 	margin: 10px;
-}
\ No newline at end of file
+}

```

Comment:
using `px` is a little rigged.  Its not a deal breaker but investigate using `em` for you sizing and positioning.  `em` is dynamic per view size which is super helpful.